src/JewellryStore/Main.java

```diff
@@ -0,0 +1,220 @@
+package JewellryStore;
+import java.io.*;
+import java.util.ArrayList;
+
+/**
+ * Description: An online jewellry store shopping system
+ * Author: L. He
+ */
+public class Main {
+    // Initialize Order
+    static Order theOrder;
+    
+    // Import BufferedReader
+    static BufferedReader keyboard = new BufferedReader(new InputStreamReader(System.in));;
+
+    public static void main(String[] args) throws IOException {
+        // Create arraylist for user's shopping bag
+        ArrayList<String> shoppingBag = new ArrayList<String>( );
+
+        // Variables
+        String necklaceMaterial;
+        String ringMaterial;
+        int pageNum = -1;
+        int productsPageNum = -1;
+        int totalQuantity = 0;
+        int totalUniqueProducts = 0;
+        double subtotal = 0;
```

Comment:
The use of the `shoppingBag` arrayList of strings is redundant with what the intended functionality of the Order object is.  As correctly designed in your diagram, the `Order` class has 1..* items and serves to maintain the list of items a user wants to purchase, along with the subtotal and total of the order.  You've taken that designed functionality out of the Order class and implemented it directly in the Main class, going against encapsulation principles and not separating the concerns of an Order from the Main program.